torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -782,13 +781,22 @@ def _check_supported_callable(fn_var):
             unimplemented(
                 f"Expected a tuple but got {args[2].python_type()}",
             )
-
         operands = args[2].unpack_var_sequence(tx)
         if not only_consist_of(args[2], (TensorVariable,)):
             unimplemented(
                 "Expect operands to be a tuple of pytrees that only consists of tensor leaves."
             )
 
+        # additional inputs check
+        if not isinstance(args[3], (ListVariable, TupleVariable)):
+            unimplemented(
+                f"Expected a tuple but got {args[3].python_type()}",
+            )
+        additional_inputs = args[3].unpack_var_sequence(tx)
+        assert (
+            len(additional_inputs) == 0
+        ), "Additional inputs are set automatically by dynamo"
```

Comment:
If additional inputs are set automatically by dynamo, what is the reason of passing an empty tuple here? Should the number of arguments match?